### PR TITLE
Correct checksum config path for Helm chart.

### PR DESCRIPTION
**Pull request by jeschkies**
**What this PR does / why we need it**:
As request in https://grafana.slack.com/archives/CEPJRLQNL/p1685453125856569

**Checklist**
- [ ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)

### CODE REVIEW

**Review by MichelHollands — APPROVED** [review]
LGTM but before merging you have to update a few more files as mentioned in the checklist.

**Comment by mkohyann** [comment]
can you add the modification of 

 **checksum/config: {{ include (print .Template.BasePath "/configmap.yaml") . | sha256sum }}**

in the files  statefulset-read.yaml & statefulset-write.yaml

**Review by trevorwhitney — APPROVED** [review]
we still need this? if so lgtm!

**Pull request merged**
